apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -357,8 +357,8 @@ suite('ActivityHandler', function() {
 
       test('passes contact name in plain text', function() {
         sinon.assert.called(sendSpy);
-        var notification = sendSpy.firstCall.thisValue;
-        assert.equal(notification.title, contactName);
+        var notification = sendSpy.firstCall;
+        assert.isTrue(notification.calledWith(contactName));
```

Comment:
Error: send(<&>, { body: "body", data: { id: 1, threadId: 1 }, icon: "sms", tag: "threadId:1" }) => { then: function () {} } is not a function